### DISCUSSION TITLE
Display trkpt track point dots

### DISCUSSION THREAD

**Discussion by 0709wiwiwi** [devemux86/cruiser#579] [Q&A] [answered]
Display trackpoint dots in a gpx  track (overlays)

Cruiser allows to promote already existing trackpoints into shaping trackpoints.
Locate on top of the navtrack | add shaping point | tap ok |= without calculation.
The previous nearest trackpoint can be  promoted into a shaping or via trkpt.
Show the exact trkpt locations by a mini track point dot display.
Similar result as offered in Garmin Basecamp or Mapsource.
![trackpoint dots](https://github.com/devemux86/cruiser/assets/54167201/ba603541-58c5-4bb6-b09a-b039229ac873)

**Comment by devemux86** [comment, marked as answer]
@0709wiwiwi thanks for the suggestion.

This will fill the map with markers and have a negative impact on performance.

**Reply by 0709wiwiwi** [reply]
**Update and simplify this idea realisation.**
- **No extra function button necessary !**
Display the (trkpt) trackpoint locations in track overlays in a most simple way.
The minimum (very narrow line) of a simple track overlays is 1 px (pd width unity).
Show trackpoints by maintining a minimum width of 2 px at the trkpt locations .
The overlay display could so be similar as in Garmin BaseCamp/MapSource.

**Comment by devemux86** [comment]
Also this is not related with the format that is only used in export / import.

First we must have a calculated or imported route.
Then it is about if we could display its path points.

As shaping or via points can only be added on routes.

**Reply by devemux86** [reply]
> a minimal basic navtrack has to exist

You probably mean a calculated or imported route.

As navtrack is just a file format to export / import routes, like the json.

**Comment by 0709wiwiwi** [comment]


**Reply by bios4** [reply]
Already existing: Settings > Routing > Appearance > Route / Tracks > Direction arrows switch

**Reply by devemux86** [reply]
Dots would be the right solution, if this feature is really necessary.

Therefore, we should first think about what is the exact need for such a feature.
